include/arch/arm/aarch64/cpu.h

```diff
@@ -28,11 +28,11 @@
 
 #define SCTLR_EL1_RES1		(BIT(29) | BIT(28) | BIT(23) | \
 				BIT(22) | BIT(20) | BIT(11))
-#define SCTLR_M_BIT		BIT(0)
-#define SCTLR_A_BIT		BIT(1)
```

Comment:
does this header get pulled in by public, application-facing headers like kernel.h or is it just used by the arch/ code? maybe we could move it to arch/arm/include area, the definitions here aren't namespaced in any particular way

Reply:
There is a bit of everything in this file including stuff used by drivers so I think we really need this in some accessible location in the include directory.